packages/contact-center/user-state/src/helper.ts

```diff
@@ -9,13 +9,36 @@ export const useUserState = ({idleCodes, agentId, cc}) => {
 
   const [elapsedTime, setElapsedTime] = useState(0);
   const [currentState, setCurrentState] = useState({});
+  let worker;
+
+  // Initialize the Web Worker using a Blob
+  const workerScript = `
+    let startTime = Date.now();
```

Comment:
Can we check if there's an updated timestamp that comes with the agent state change event? Remember discussing something like this during SDK implementation with Ravi. 

If so, we can use that as start time.